process_report/process_report.py

```diff
@@ -165,6 +172,11 @@ def main():
         required=False,
         help="Name of csv file listing previously billed PIs",
     )
+    parser.add_argument(
```

Comment:
You should specify a type here as int, this way you don't need to typecast it later in the program.